ui/raidboss/data/06-ew/trial/endsinger.txt

```diff
@@ -0,0 +1,100 @@
+# Endsinger Normal
+# -p "662C:13.7" "6B58:1000" "6B5B:2000"
+# -it 'The Endsinger' 'Kakodaimon'
+# -oc 'The Endsinger' 'Kakodaimon'
+# -ii 6281 6C61 6660 6661 6C6A 664B 6653 6643 67FA 6642 6648 67FB 6655
+
+# Not sure if/where P1 loops
```

Comment:
I have this:
```
490.4 "--untargetable--"
499.7 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
501.8 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
507.8 "--targetable--"
514.0 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
520.0 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
528.8 "Pharmakon" sync / 1[56]:[^:]*:Feather of Despair:664C:/
539.2 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
549.4 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
559.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
576.8 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
583.2 "--sync--" sync / 1[56]:[^:]*:Doomed Stars:662D:/
590.2 "Stellar Collision" sync / 1[56]:[^:]*:Doomed Stars:662E:/
596.0 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
603.1 "--untargetable--"
612.4 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
614.6 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
620.6 "--targetable--"
626.7 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
632.7 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
641.5 "Pharmakon" sync / 1[56]:[^:]*:Feather of Despair:664C:/
642.8 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
651.9 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
662.1 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
671.7 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
672.8 "Liturgy of the Bell" sync / 1[56]:[^:]*:Liturgic Bell:6508:/
689.6 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
696.0 "--sync--" sync / 1[56]:[^:]*:Doomed Stars:662D:/
703.0 "Stellar Collision" sync / 1[56]:[^:]*:Doomed Stars:662E:/
708.8 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
715.9 "--untargetable--"
725.2 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
727.3 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
733.3 "--targetable--"
739.5 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
745.5 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
754.3 "Pharmakon" sync / 1[56]:[^:]*:Feather of Despair:664C:/
755.6 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
```

It seems possible that 745.5 loops back to 632.7?

Reply:
I think you're right. It would be highly unusual for a normal-mode trial to have a phase 1 loop, but definitely not impossible.

Reply:
Oh no, I take this back, I'm not sure what's going on here.  I've added in some more lines above, but when I run it through test_timeline, I get this:

```
    Missed sync: Aporrhoia at 436.6 (last seen at 285.68)
    Missed sync: Elenchos at 450.8 (last seen at 417.3)
    Missed sync: Elenchos at 563.5 (last seen at 417.3)
```

In particular, I don't see the double Elenchos at 450.8 or 563.5, and I'm missing that Aporrhoia entirely? I don't see it in make_timeline either on my log.  (Happy to share it if it's more helpful than just the above make_timeline output.)

Reply:
I'm so sorry you had a P1 last that long... The longest I've had was only about 600 seconds.

It might be a two-pattern loop, e.g. death => hubris is one pattern, and death => elenchos is the other? This is pretty beyond my knowledge of timelines to figure out/set up though.

Reply:
> I'm so sorry you had a P1 last that long...

Me tooooo

> It might be a two-pattern loop, e.g. death => hubris is one pattern, and death => elenchos is the other?

Yeah, this is plausible.  Feel free to leave this as a TODO for now and with more data we can fix it.  When I left the initial comment I thought maybe this could be fixed up more easily, but now it seems more complicated.